### PR TITLE
Assignment 2b and Assignment 3

### PR DESCRIPTION
**Pull request by LuisLopezPi**

### CODE REVIEW

File: Assignment2/LuisLopezPi-fromJSONtoJSON-LD.jsonld [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+		"name": "fiupm:name",
+		"hometown": "fiupm:hometown",
+		"age": "fiupm:age",
+		"almaMater": "fiupm:alma mater",
+		"mother": "fiupm:mother",
+		"father": "fiupm:father",
+		"subjects": "fiupm:subjects",
+		"description": "subjects:description",
+		"nameS": "subjects:nameS",
+		"teacher": "subjects:teacher"
```
Comment (paoespinozarias) [inline Assignment2/LuisLopezPi-fromJSONtoJSON-LD.jsonld:13]: These subject attribute definitions work, but semantically they aren't ok. The best way to define them might be defining a new prefix for the subject concept and then use it to define their attributes.
You don't need to change them, but remember to provide good definitions for the future.

File: Assignment2/LuisLopezPi-fromJSONtoJSON-LD.jsonld [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+		"teacher":"Oscar Corcho",
+		"description":"El objetivo de esta asignatura es proporcionar la base teórica y práctica sobre los fundamentos metodológicos y tecnológicos implicados en la generación y publicación de vocabularios y datos en la Web de datos enlazados, así como en forma de grafos de conocimientos"
+		},
+
+		{
+		"nameS":"Administración de Sistemas informáticos",
+		"teacher":"Francisco Rosales García",
+		"description":"En esta asignatura se muestran los fundamentos de la administración de sistemas informáticos"
+		}]
+}
```
Comment (paoespinozarias) [inline Assignment2/LuisLopezPi-fromJSONtoJSON-LD.jsonld:36]: This file is ok, well done.

File: Assignment2/LuisLopezPi-fromRDFtoJSON-LD.jsonld [hunk 3]
```diff
@@ -0,0 +1,59 @@
+{
+	"@context":{
+	"Assignment2":"http://Assignment2.com/SemanticWeb/",
+	"Class01":"Assignment2:Class01",
+	"Sensor029":"Class01:Sensor029",
+	"Computer101":"Class01:Computer101",
+	"Measurement8401":"Sensor029:Measurement8401",
+	"User10A":"Computer101:User10A",
```
Comment (paoespinozarias) [inline Assignment2/LuisLopezPi-fromRDFtoJSON-LD.jsonld:8]: You are defining them as "@type" on your data, thus you don't need to define them here.

File: Assignment2/LuisLopezPi-fromRDFtoJSON-LD.jsonld [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+	"hasTemperature":"Assignment2:hasTemperature",
+	"atTime":"Assignment2:atTime",
+	"hasOwner":"Assignment2:hasOwner",
+	"hasName":"Assignment2:hasName"
+	},
+	"@id":"http://Assignment2.com/SemanticWeb/Class01/",
+	"@type":"Assignment2:Class01",
+	"includes":[
+		{	
+			"Sensor029":{
```
Comment (paoespinozarias) [inline Assignment2/LuisLopezPi-fromRDFtoJSON-LD.jsonld:20]: You don't need to define "Sensor029". It isn't an attribute, it is a resource, thus I suggest you to remove it. Remember that when you define a resource, you only have to use curly braces and define inside all the resource attributes. However, when you have to specify a relationship between resources is when you have to define it as an attribute, for example as you did with the attribute "hasMeasurement" for defining the relationship between Sensor029 and Measurment8401. Note that "hasMeasurement" must be defined in the @context as you have done.
This comment also applies for "Measurement8401", "Computer101", and "User10A".

**Review by paoespinozarias — CHANGES_REQUESTED** [review]

**Review by paoespinozarias — COMMENTED** [review]
Assignment 2b is ok. Please, @ocorcho review Assignment 3

**Comment by ocorcho** [comment]
Assignment 3 is ok, but you have to rename the files as suggested (inside your own folder/directory) so that there are no conflicts.

**Comment by ocorcho** [comment]
Given that the changes requested by me 6 days ago have not been dealt with, we are closing this pull request.
If you want this to be considered further, please contact us by e-mail.

**Pull request closed without merging**